packages/inter-protocol/src/reserve/assetReserveKit.js

```diff
@@ -166,6 +198,67 @@ export const prepareAssetReserveKit = async (
           facets.helper.writeMetrics();
         },
       },
+      withdrawalFacet: {
+        Withdraw() {
+          /** @type {OfferHandler<Promise<string>>} */
+          const handler = async seat => {
+            try {
+              guard.assert();
+              const { state } = this;
+              const { helper } = this.facets;
+              const { want } = seat.getProposal();
+              const withdrawMap = new Map();
+              const depositMap = new Map();
+
+              const wantEntries = Object.entries(want);
+              for (const [keyword, amount] of wantEntries) {
```

Comment:
To make it obvious that `wantEntries` is used only once, I suggest inlining it:

```suggestion
              for (const [keyword, amount] of entries(want)) {
```

with `const { entries } = Object` at some higher scope.

Reply:
That code is now removed.